src/assert.bash

```diff
@@ -185,12 +185,14 @@ assert_failure() {
 assert_output() {
```

Comment:
Could you update the function comment too? For example:

```diff
 # Options:
 #   -p, --partial - partial matching
 #   -e, --regexp - extended regular expression matching
+#   - - read expected output from the standard input
 # Arguments:
-#   $1 - [=STDIN] expected output
+#   $1 - expected output
 # Returns:
 #   0 - expected matches the actual output
 #   1 - otherwise
```